agent/util-scripts/pbench-move-results

```diff
@@ -83,12 +99,17 @@ done
 if [[ -z "${controller}" ]]; then
     controller=${_pbench_full_hostname}
     if [[ -z "${controller}" ]]; then
-        error_log "Missing controller name (should be \"hostname -f\" value)"
+        error_log "Missing required controller name (should be \"hostname -f\" value)"
```

Comment:
Actually, there is no reason why the controller name "should" be the hostname....  Moreover, what we're trying to report here is that we have _no_ value for `controller` because `hostname -f` produced a blank, so calling for it to be set to that value isn't helpful, either.

How about something like "no hostname configured, please specify --controller" or "--controller is required, because no hostname is configured".